src/main/java/org/folio/rest/impl/InventoryHelper.java

```diff
@@ -34,13 +34,34 @@
 import static me.escoffier.vertx.completablefuture.VertxCompletableFuture.allOf;
 import static org.folio.orders.utils.HelperUtils.calculateInventoryItemsQuantity;
 import static org.folio.orders.utils.HelperUtils.encodeQuery;
+import static org.folio.orders.utils.HelperUtils.groupLocationsById;
 import static org.folio.orders.utils.HelperUtils.handleGetRequest;
 import static org.folio.orders.utils.HelperUtils.verifyAndExtractBody;
 import static org.folio.rest.impl.AbstractHelper.ID;
 
 public class InventoryHelper {
 
-  public static final String ON_ORDER_ITEM_STATUS = "On order";
+  static final String INSTANCE_SOURCE = "source";
+  static final String INSTANCE_TITLE = "title";
+  static final String INSTANCE_EDITIONS = "editions";
+  static final String INSTANCE_STATUS_ID = "statusId";
+  static final String INSTANCE_TYPE_ID = "instanceTypeId";
+  static final String INSTANCE_PUBLISHER = "publisher";
+  static final String INSTANCE_DATE_OF_PUBLICATION = "dateOfPublication";
+  static final String INSTANCE_PUBLICATION = "publication";
+  static final String INSTANCE_IDENTIFIER_TYPE_ID = "identifierTypeId";
+  static final String INSTANCE_IDENTIFIERS = "identifiers";
+  static final String INSTANCE_IDENTIFIER_TYPE_VALUE = "value";
+  static final String HOLDING_INSTANCE_ID = "instanceId";
+  static final String HOLDING_PERMANENT_LOCATION_ID = "permanentLocationId";
+  static final String ITEM_HOLDINGS_RECORD_ID = "holdingsRecordId";
+  static final String ITEM_STATUS = "status";
+  static final String ITEM_STATUS_NAME = "name";
+  static final String ITEM_STATUS_ON_ORDER = "On order";
+  static final String ITEM_MATERIAL_TYPE_ID = "materialTypeId";
+  static final String ITEM_PERMANENT_LOAN_TYPE_ID = "permanentLoanTypeId";
+  static final String ITEM_PURCHASE_ORDER_LINE_IDENTIFIER = "purchaseOrderLineIdentifier";
```

Comment:
Unless there's a reason for this, can you please make these private so their consistent with the other constants below?

Reply:
I decided to define Inventory objects' properties as constants just to use them also in unit tests to verify expected content of the inventory records. This is why all of them are package private.